steps/mountbootwiminbuildwinpeisoonwindowsworker/README.md

```diff
@@ -0,0 +1 @@
+Mounts `C:\attuneautomationworker\build-winpe-iso\winpe_staging\SOURCES\BOOT.WIM` to `C:\attuneautomationworker\build-winpe-iso\WinPE_BootImageDir` using `wimmountrw` on WSL2.
```

Comment:
Why is this line so long?

Reply:
@jarrodchesney , should I make them shorter by for example in this case rewrite the comment as?
```
Relative to the `C:\attuneautomationworker\build-winpe-iso\` directory, mounts `winpe_staging\SOURCES\BOOT.WIM` to `build-winpe-iso\WinPE_BootImageDir` using `wimmountrw` on WSL2.
```